src/garage/sampler/on_policy_vectorized_sampler.py

```diff
@@ -23,6 +23,19 @@ class OnPolicyVectorizedSampler(BatchSampler):
         env (gym.Env): A gym/akro env object
         n_envs (int): Number of parallel environments used for sampling.
 
+    In obtain_samples(), N paths are generated. Each path is a dictionary,
+    with keys and values as following:
+    - observations : numpy.ndarray with shape [Batch, *obs_dims]
```

Comment:
again, these will render poorly on readthedocs.io. please just don't exploit fixed-width in docstrings.